app/script/backup/BackupRepository.js

```diff
@@ -138,6 +138,18 @@ z.backup.BackupRepository = class BackupRepository {
       );
     });
 
+    const mapEntityDataTypes = entity => {
+      const mappedFields = ['otr_key', 'sha256'];
+      if (entity.data) {
+        mappedFields.forEach(field => {
+          if (entity.data[field]) {
+            entity.data[field] = Uint8Array.from(Object.values(entity.data[field]));
```

Comment:
Please check that Object.values is supported by our browser range. I recall that we had an issue with this and outdated wrappers in the wild at some point in the past.

Reply:
Good call. I will check.